src/coreclr/tools/Common/Compiler/ProcessLinkerXmlBase.cs

```diff
@@ -180,26 +198,28 @@ protected virtual void ProcessTypes(ModuleDesc assembly, XPathNavigator nav, boo
                 if (!ShouldProcessElement(typeNav))
                     continue;
 
-                string fullname = GetFullName(typeNav);
+                string fullName = GetFullName(typeNav);
```

Comment:
Not blocking, but if at some point we try to unify this file with illink, one of the things we would do is diffing the two files. These changes would make for annoying diffs. Without this rename, the lines would be identical here (modulo the unique-linker-snowflake formatting rules, but those can be gotten rid of in an automated fashion).

The renames also add extra overhead to this review because there's more things to look at. They're usually better done in separate PRs, if done at all.

Reply:
Removed these changes